server/blob/blob-export-api/src/main/java/org/apache/james/blob/export/api/BlobExportMechanism.java

```diff
@@ -29,14 +29,33 @@ public BlobExportException(String message, Throwable cause) {
         }
     }
 
+    enum BlobFileExtension {
```

Comment:
I think we don't want to restrict the use of extensions for a small subset of prossibilities. IMO this should be a class so that the caller can pass the value he/she wants without being restricted.

Reply:
IMO this is `FileExtension`, nothing specific to blob store.